cpp/src/io/comp/nvcomp_adapter.cu

```diff
@@ -94,10 +94,10 @@ void update_compression_results(device_span<size_t const> actual_output_sizes,
     [] __device__(auto const& results) { return results.status != compression_status::SKIPPED; });
 }
 
-size_t skip_unsupported_inputs(device_span<size_t> input_sizes,
-                               device_span<compression_result> results,
-                               std::optional<size_t> max_valid_input_size,
-                               rmm::cuda_stream_view stream)
+void skip_unsupported_inputs(device_span<size_t> input_sizes,
```

Comment:
Not sure why the previous version of this function was both skipping *and* returning the sum.
This version is better.

Reply:
It's because I didn't want to add a function that would just be a wrapper around `thrust::reduce`.
The function now wraps TWO `thrust::reduce`s so it's okay 🙈